drivers/dai/intel/dmic/dmic_nhlt.c

```diff
@@ -347,49 +513,8 @@ int dai_dmic_set_config_nhlt(struct dai_intel_dmic *dmic, const void *bespoke_cf
 
 		val = *(uint32_t *)p;
 		out_control[n] = val;
-		bf1 = FIELD_GET(OUTCONTROL_TIE, val);
-		bf2 = FIELD_GET(OUTCONTROL_SIP, val);
-		bf3 = FIELD_GET(OUTCONTROL_FINIT, val);
-		bf4 = FIELD_GET(OUTCONTROL_FCI, val);
-		bf5 = FIELD_GET(OUTCONTROL_BFTH, val);
-		bf6 = FIELD_GET(OUTCONTROL_OF, val);
-		bf7 = FIELD_GET(OUTCONTROL_IPM, val);
-		bf8 = FIELD_GET(OUTCONTROL_TH, val);
-		LOG_INF("dmic_set_config_nhlt(): OUTCONTROL%d = %08x", n, out_control[n]);
-		LOG_INF("  tie=%d, sip=%d, finit=%d, fci=%d", bf1, bf2, bf3, bf4);
-		LOG_INF("  bfth=%d, of=%d, ipm=%d, th=%d", bf5, bf6, bf7, bf8);
-		if (bf5 > OUTCONTROL_BFTH_MAX) {
-			LOG_ERR("dmic_set_config_nhlt(): illegal BFTH value");
-			return -EINVAL;
-		}
```

Comment:
Won't we miss these error codes? Because if I think correctly, now we will move on to configuring DMIC, despite the fact that the received configuration is not correct.